R/getClusterAnalysis.R

```diff
@@ -207,7 +220,7 @@ getClusterAnalysis = function(data, num.features, method, par.vals, random.seed,
       proc.data = as.data.frame(cbind(pca.data$x, cluster = kernel.cluster@.Data))
       proc.data$cluster = as.factor(proc.data$cluster)
       ##ggscatter
-      kernel.plot = ggscatter(data = proc.data, x = "PC1", y = "PC2",
+      kernel.plot = ggpubr::ggscatter(data = proc.data, x = "PC1", y = "PC2",
```

Comment:
why did you add the `::` operator?
You should use instead:
`@importFrom ggpubr ggscatter`

Reply:
I get an error when importing packe ggpubr or functiion ggscatter into the namespace. Instead I put it in DESCRIPTION

Reply:
I get an error when importing packacke ggpubr or functiion ggscatter into the namespace. Instead I put it in DESCRIPTION